client/cody/src/editor/vscode-editor.ts

```diff
@@ -10,6 +10,21 @@ import {
 const SURROUNDING_LINES = 50
 
 export class VSCodeEditor implements Editor {
+    public getWorkspaceRootPath(): string | null {
+        const uri = vscode.window.activeTextEditor?.document.uri
+        if (uri) {
+            const wsFolder = vscode.workspace.getWorkspaceFolder(uri)
+            if (wsFolder) {
+                return wsFolder.uri.fsPath
+            }
+        }
+
+        if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length >= 1) {
```

Comment:
More JavaScripting to rely on falsyness of 0, if eslint doesn't mind.

```suggestion
        if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length) {
```